pinot-core/src/main/java/org/apache/pinot/core/operator/blocks/results/AggregationResultsBlock.java

```diff
@@ -69,7 +84,14 @@ public DataSchema getDataSchema(QueryContext queryContext) {
     ColumnDataType[] columnDataTypes = new ColumnDataType[numColumns];
     for (int i = 0; i < numColumns; i++) {
       AggregationFunction aggregationFunction = _aggregationFunctions[i];
-      columnNames[i] = aggregationFunction.getColumnName();
+      String columnName = aggregationFunction.getResultColumnName();
```

Comment:
I feel this is a very important change to call out, as I'm likely lacking context to understand all the implications.  I know that at least one unit test failed initially due to this change where the resulting columns were named `count_star` previously, vs `count(*)` now.  This may be an undesirable change.

Reply:
I checked and we are not using the column name returned from the server in `AggregationDataTableReducer`. You may verify that by reverting the changes in this class and see if all the new tests still pass. If that is the case, we can also consider dropping the changes in this class to simplify the changes.

Reply:
I had mistakenly assumed that this change was needed to ensure that non-group-by filtered aggs had their result column names accurately reflected.  That appears to not be the case having confirmed that tests still pass after removing these changes.

Is it at all problematic that this section of code would be "out of sync" with other sections of code as it pertains to column naming?